spec/controllers/to_dos_controller_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+RSpec.describe ToDosController, type: :controller do
+
+    let(:to_do) {FactoryGirl.create(:to_do)}
```

Comment:
I don't think you have a linter yet, but best practice is to put spaces inside a block, like: `{ FactoryGirl.create(:to_do) }`.